core/api/views/projects.py

```diff
@@ -41,3 +45,15 @@ class ProjectListView(mixins.ListModelMixin, generics.GenericAPIView):
     )
     def get(self, request, *args, **kwargs):
         return self.list(request, *args, **kwargs)
+
+
+class ProjectCreateView(mixins.CreateModelMixin, generics.GenericAPIView):
+    """
+    API endpoint that allows projects submission
+    """
+
+    queryset = Project.objects.select_related("submission")
+    serializer_class = ProjectDetailsSerializer
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
```

Comment:
This more of a personal preference thing, so feel free to disregard. But you might want to consider combining the list and get into a single `ViewSet` instead of having separate `View` classes for each action. https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset